app/views/qa/show.html.erb

```diff
@@ -31,7 +31,7 @@
       <% elsif single_wrapped_question? %>
         <% checkboxes = render "govuk_publishing_components/components/checkboxes", {
           name: "#{current_facet['facet']['key']}[]",
-          heading: "Select all that apply",
+          heading: current_facet["question"],
```

Comment:
Doesn't this repeat the heading twice?

What the docs seem to prefer is using the heading _as_ the legend, rather than just repeating it, if that makes sense.

Reply:
@ChrisBAshton - which docs are you referring to?

Reply:
https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/with_legend_as_page_heading

Reply:
What's the value in having the visually-hidden class in this example?

Reply:
@ChrisBAshton - we cannot use this as the question is in a title component ( https://github.com/alphagov/finder-frontend/blob/29717a425b8a480738bdbbda51bba2fc2ed6ca83/app/views/qa/show.html.erb#L12)

@leenagupte - we have to hide it apparently rather than removing:

> When more than one checkbox is shown they are wrapped in a fieldset element, which requires a legend. This must be supplied to the component using the heading option.

ref: 
- https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/with_multiple_checkboxes
- https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/with_a_hidden_heading

Reply:
I imagined we'd have to do something like this:

```html
<% if multiple_question? %>
  <!-- Let the checkboxes component render the title for us -->
  <% checkboxes = render "govuk_publishing_components/components/checkboxes", {
    name: "#{current_facet['facet']['key']}[]",
    heading: current_facet["question"],
    items: options
  } %>
<% else %>
  <!-- Render a separate title -->
  <%= render "govuk_publishing_components/components/title", {
    title: current_facet["question"],
  } %>
  <!-- Render the question here -->
<% endif %>
```

I appreciate this is a more substantial change than merely passing an argument to a component...!

Reply:
Surely we always need to render the checkboxes?

Reply:
We always need to render the checkboxes (or radio buttons, or whatever the input may be) but we don't always need to render the title component, because sometimes the input component will render the title for us in an accessible way. Code sample is shortened for clarity ^